src/twisted/python/test/test_url.py

```diff
@@ -802,11 +802,11 @@ def test_technicallyTextIsIterableBut(self):
 
 class URLDeprecationTests(SynchronousTestCase):
     """
-    L{twisted.python.constants} is deprecated.
+    L{twisted.python.url} is deprecated.
```

Comment:
I'm guessing @hawkowl couldn't help herself and cleaned up a mistake I introduced.